sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/TableCatalog.java

```diff
@@ -230,6 +235,23 @@ default Table createTable(
     return createTable(ident, CatalogV2Util.v2ColumnsToStructType(columns), partitions, properties);
   }
 
+  /**
+   * Create a table in the catalog.
+   *
+   * @param ident a table identifier
+   * @param tableInfo Information about the table.
```

Comment:
Minor: `Information` -> `information` to be consistent with the rest of the class?